agent/engine/docker_task_engine_test.go

```diff
@@ -4229,7 +4246,12 @@ func TestPullContainerManifest(t *testing.T) {
 				versioned.EXPECT().
 					PullImageManifest(gomock.Any(), "myimage", nil).
 					Return(
-						registry.DistributionInspect{Descriptor: ocispec.Descriptor{Digest: testDigest}},
+						registry.DistributionInspect{
+							Descriptor: ocispec.Descriptor{
+								MediaType: "application/vnd.docker.distribution.manifest.v2+json",
```

Comment:
nit: maybe we should define a constant for v2 schema and use it here.

Reply:
updated